src/examples.md

```diff
@@ -2346,7 +2346,23 @@ _This miniapp has only a parallel ([contact-patch-test.cpp](https://github.com/m
 <div style="clear:both;"/></div>
 <br></div>
 
+<div id="contact" markdown="1">
+## Contact miniapp
 
+<a href="../img/gallery/workshop25/contact.mp4" target="_blank">
+<img class="floatright" width="250" src="../img/gallery/workshop25/contact.png">
+</a>
+
+This [miniapp](https://github.com/mfem/mfem/blob/master/miniapps/contact/contact.cpp) demonstrates how to use MFEM in combination with [Tribol](https://github.com/LLNL/Tribol)
+ to simulate frictionless contact between deformable solids. The contact gap function and its Jacobian are computed using Tribol’s mortar segment-to-segment method.  An **Interior-Point (IP)** optimization solver is used to solve an inequality-constrained minimization problem enforcing a non-penetration condition. Linear systems arising in the IP iterations are solved using **Conjugate Gradient (CG)** preconditioned with the [**AMG with Filtering (AMGF)**](https://arxiv.org/abs/2505.18576) solver.
```

Comment:
Can you add a link to a paper describing the details of the contact method you are using?